src/python/pants/backend/codegen/thrift/java/thrift_defaults.py

```diff
@@ -25,12 +25,21 @@ def register_options(cls, register):
              help='The default language to generate for java_thrift_library targets.')
     register('--rpc-style', type=str, advanced=True, default='sync',
              help='The default rpc-style to generate for java_thrift_library targets.')
+    register('--default-java-namespace', type=str, advanced=True, default=None,
+             help='The default Java namespace to generate for java_thrift_library targets.')
+    register('--namespace-map', type=dict, advanced=True, default={},
+             help='The default namespace map to generate for java_thrift_library targets, {old: new}.')
+    register('--include-paths', type=list, advanced=True, default=[],
```

Comment:
@stuhood what do you think about omitting these from defaults entirely? i just added these as cargo-cult.